examples/src/components/AreaChartExample.jsx

```diff
@@ -41,6 +41,7 @@ export class AreaChartExample extends Component {
         return (
             <div style={{ height: 300 }} className="s-area-chart">
                 <AreaChart
+                    {...this.props}
```

Comment:
this spreading of all props is not best practice. It hides which props are used by component

Reply:
Updated. Replaced 
<AreaChart
                    {...this.props}
with
<AreaChart
                    onExportReady={this.props.onExportReady}